Reach/Payment/Model/DutyCalculator.php

```diff
@@ -342,6 +352,21 @@ protected function getHsCode($sku)
         return null;
     }
 
+        /**
```

Comment:
while you're  in these files, can you fix the spacing on this line? Small thing haha